visualizations/frontend/filters/TTagsSelector.vue

```diff
@@ -339,6 +341,21 @@ export default {
       }
       return false;
     },
+    displayValue(value, key) {
+      const row = this.rows.find((r) => {
+        if (value) {
+          const v = r[this.tValueColumn];
+          return v.value === value;
+        } else {
+          const k = r[this.tKeyColumn];
+          return k.value === key;
+        }
+      });
+      if (row?.DISPLAYVALUE) {
+        return row.DISPLAYVALUE.value;
+      }
+      return value ? value : key;
+    },
```

Comment:
At first, it was hard for me to read and understand this code, I was thinking how can we improve it, and I came up with something like this:

```suggestion
    displayValue(value, defaultValue = "") {
      if (!value) {
        return "";
      }

      const row = this.rows.find((row) => {
          const columnValue = row[this.tValueColumn];

          return columnValue.value === value;
        });

      return row?.DISPLAYVALUE?.value || defaultValue;
    },
    
    displayKey(key, defaultKey = "") {
      if (!key) {
        return "";
      }

      const row = this.rows.find((row) => {
        const columnKey = row[this.tKeyColumn];
      
        return columnKey.value === key;
      });

      return row?.DISPLAYKEY?.value || defaultKey;
    },
```

It makes the API clearer IMHO, instead of having a `value` or `key` signature, we can have dedicated methods for each. It's more verbose but it increase the simplicity.

Please, let me know what are your thoughts about it.